search/api_test.py

```diff
@@ -60,6 +65,24 @@ def search():
     return get(url).json()['hits']
 
 
+def mappings():
```

Comment:
can you rename this to `current_es_mappings()` or `get_current_es_mappings()`? this snippet below is just a little confusing as it is:

    mapping = mappings()
    nodes = list(traverse_mapping(mapping))